### PR TITLE
[WIP] Full Audit, ref issue #1706

### LINKED ISSUE #1706: Full audit: Permissions matrix for Cloud Big Data v2
https://github.com/rackerlabs/rackspace-how-to/blob/master/content/cloud-big-data/detailed-permissions-matrix-for-cloud-big-data-v2.md

- [x] SME check: Update any out of date or inaccurate technical content with the help of a SME
- [x] Editor check: Incorporate edits from Kelly Holcomb
- [x] Quality Checklist: Perform a quality audit of the article using the quality checklist
- [x] Update audit_date: Update the audit_date metadata in the article with the date of the full audit's completion

### PR DESCRIPTION
**Pull request by nimbinatus**
proofed before SME review

Fixes #1706

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-424d54fd43/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cbc5c2c6a7/how-to/)

**Comment by rnirmal** [comment]
+1

**Comment by nimbinatus** [comment]
@kmsholcomb , can you add this to your editing queue? Thanks!

**Comment by kmsholcomb** [comment]
Will do, @nimbinatus.

**Comment by kmsholcomb** [comment]
Hi, @nimbinatus. My very few edits are in the attached Word doc (and PDF copy of it). Please let me know if you have any questions. Thanks!

[Detailed Permissions Matrix for Cloud Big Data v2_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/664109/Detailed.Permissions.Matrix.for.Cloud.Big.Data.v2_editsKH.pdf)
[Detailed Permissions Matrix for Cloud Big Data v2_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/664108/Detailed.Permissions.Matrix.for.Cloud.Big.Data.v2_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3543e84630/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-07b550b8f1/how-to/)

File: content/cloud-big-data/detailed-permissions-matrix-for-cloud-big-data-v2.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ The Cloud Big Data v2 permissions matrix displays specific permissions for the f
...
 
-The matrix displays the Cloud Big Data v2 methods, their corresponding RESTful API commands, and the RBAC roles that are supported.
+The matrix displays the Cloud Big Data v2 methods grouped by category, their corresponding RESTful API commands, and the RBAC roles that are supported.
 
 ### Credentials
 
 Method | API action | Role | Description
 --- | --- | --- | ---
-List all credentials | `GET /v2/{tenantId}/credentials` | **Admin<br/>Creator<br/>Observer** | Lists all user credentials
+List all credentials | `GET /v2/{tenantId}/credentials` | **Admin<br/>Creator<br/>Observer** | Lists all user credentials.
```
Comment (DonNateR) [inline content/cloud-big-data/detailed-permissions-matrix-for-cloud-big-data-v2.md]: Change all instances of `<br/>` to a comma, in order to keep RBAC tables consistent across the board.

Comment (nimbinatus) [inline content/cloud-big-data/detailed-permissions-matrix-for-cloud-big-data-v2.md]: Changes made

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-fb98ba6ec5/how-to/). :bow:

**Pull request merged**